packages/mui-material/src/useMediaQuery/useMediaQuery.ts

```diff
@@ -30,7 +30,6 @@ export interface Options {
    * This option is kept for backwards compatibility and has no longer any effect.
    * It's previous behavior is now handled automatically.
    */
-  // TODO: Deprecate for v6
```

Comment:
Nop this has use cases, https://mui.com/material-ui/react-use-media-query/#client-side-only-rendering, see the diff above.